code-challenges.test.js

```diff
@@ -37,6 +73,32 @@ const arrayOfWords2 = ["Mango", "Cherry", "Apricot", "Blueberry", "Peach"]
 const letterE = "e"
 // Expected output: ["Cherry", "Blueberry", "Peach"]
 
+const found = (array) => {
+    for (let i = 0; i < array.length; i++){
+    array.find(element => 'a')
+    }
+    return array
+}
```

Comment:
It appears that after all the hard work you are doing to iterate, your function is just returning the parameter.